### PR TITLE
[fix] ref.focus undefined && newContents component undefined

### PR DESCRIPTION
**Pull request by mqtik**
When we add/remove multiple images, the editor can't resolve if ref.focus is not defined. This also occurs when removing a image and adding another one, the newContents.component returns undefined.

### CODE REVIEW

File: src/CNRichTextEditor.js [hunk 1]
```diff
@@ -223,7 +223,7 @@ class CNRichTextEditor extends Component {
       };
 
       let newConents = value;
-      if (newConents[index - 1].component === 'text') {
+      if (ewConents[index - 1] && newConents[index - 1].component === 'text') {
```
Comment (imnapo) [inline src/CNRichTextEditor.js]:
```suggestion
      if (newConents[index - 1] && newConents[index - 1].component === 'text') {
```

**Comment by mqtik** [comment]
I also added a commit to resolve images that don't have size properties.
It gets the size property by using Image.getSize (as it does when inserting), and updates.

**Comment by mqtik** [comment]
This library is going to be amazing.

I'm testing it hard and will pull some more request!

**Comment by mqtik** [comment]
Also added conditionals on Convertors to avoid app freezes and still render the content if something goes wrong

**Pull request merged**